src/containers/Permissions.jsx

```diff
@@ -50,19 +51,22 @@ const Permissions = () => {
           {queryResultApps.data
             .concat(queryResultKonnectors.data)
             .sort((a, b) => a.slug.localeCompare(b.slug))
-            .map(appOrKonnector => (
-              <div key={appOrKonnector.name}>
-                <Link to={'/permissions/' + appOrKonnector.slug}>
-                  <ListItem button>
-                    <ListItemIcon>
-                      <AppIcon app={appOrKonnector} />
-                    </ListItemIcon>
-                    <ListItemText primary={appOrKonnector.name} />
-                  </ListItem>
-                </Link>
-                <Divider />
-              </div>
-            ))}
+            .map(
+              appOrKonnector =>
+                !toNotDisplay.includes(appOrKonnector.name) && (
```

Comment:
You should not rely on the `name`. `name` is something that can change and can be overwritten depending on the Cozy's context. Use the `slug` attribute instead.